proxy-node-shared/src/main/java/uk/gov/ida/notification/logging/TranslatorSigningLoggerHelper.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.ida.notification.logging;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+
+public class TranslatorSigningLoggerHelper {
+
+    private static final Logger log = LoggerFactory.getLogger(TranslatorSigningLoggerHelper.class);
```

Comment:
By convention `static final` fields have UPPER_CASE_NAMES

Reply:
Sure but that's the way it has been done in other logger classes by other devs including yourself, so rather keep it consistent for now.